.github/actions/internal-camunda-chart-tests/action.yml

```diff
@@ -458,3 +524,164 @@ runs:
                 exit 1
               fi
               echo "✅ The cluster meets all the expected criteria."
+
+        - name: 🧪 C8-SM-CHECKS - Run Deployment Check
+          if: ${{ inputs.enable-c8sm-deployment-check == 'true' }}
+          shell: bash
+          env:
+              NAMESPACE: ${{ inputs.test-namespace }}
+              HELM_DEPLOYMENT_NAME: ${{ inputs.test-release-name }}
+          run: |
+              set -euo pipefail
+
+              echo "🔍 Running deployment check..."
+
+              # Determine required containers based on what's enabled and Camunda version
+              # For Camunda 8.8+, zeebe components are unified under 'orchestration'
+              CAMUNDA_MAJOR_VERSION=$(echo "${{ inputs.camunda-version }}" | cut -d. -f1)
+              CAMUNDA_MINOR_VERSION=$(echo "${{ inputs.camunda-version }}" | cut -d. -f2)
+
+              if [[ "$CAMUNDA_MAJOR_VERSION" -ge 8 && "$CAMUNDA_MINOR_VERSION" -ge 8 ]]; then
+                # Camunda 8.8+ uses 'zeebe' instead of 'zeebe,zeebe-gateway'
+                REQUIRED_CONTAINERS="connector,optimize,zeebe"
+              else
+                # Older versions use separate zeebe and zeebe-gateway
+                REQUIRED_CONTAINERS="connector,optimize,zeebe,zeebe-gateway"
+              fi
```

Comment:
there's no PR description, so no idea what the plan is.
Do you want to backport it to 8.7 and earlier?

Otherwise this could be simplified. Generally could be since you don't have to care about the version within the branch you are since `.camunda-version` is set to one version.
Like you wouldn't call this action from this ref with a different version than what the code is defined for.

Reply:
like safe yourself the versioning and just say
`REQUIRED_CONTAINERS=...`

Reply:
what I mean is that you can save yourself the whole versioning logic because the test workflow is always related to the version of the branch.
Like you wouldn't trigger this action with a 8.7 version from the 8.8 branch and vice versa.